config/environments/production.rb

```diff
@@ -19,7 +19,7 @@
     host: Figaro.env.domain_name,
     protocol: 'https',
   }
-  config.action_mailer.asset_host = Figaro.env.mailer_domain_name
+  config.action_mailer.asset_host = Figaro.env.asset_host
```

Comment:
I don't think this does anything. I didn't see any requests to our servers or Cloudfront when opening up email in QA.

Reply:
Good point. We should either make it required, or fall back to serving assets from nginx.